help/de/Help_ch_Account-Actions.xml

```diff
@@ -917,15 +917,22 @@
           <guimenu>Datei</guimenu> in der Liste der zuletzt geöffneten Dateien zu finden; der erste
           Eintrag mit der Nummer 1 ist der Name der aktuell verwendeten Datei.
         </para>
+<!-- FIXME source: new item! -->
+        <note>
+          <para>Ab <application>&appname;</application> Version 5.0 wird für die Aktualisierung der Wertpapiere
+            das Kommando <code>gnucash-cli --quotes get &lt;gnucash-file-name&gt;</code>
+            verwendet.
```

Comment:
Das klappt jetzt schon, also dieser Aufruf in den Text und die alte Form in eine Fußnote: Bis einschließlich Gnucash 4.x …

Reply:
ok. Ich habe die beiden Abschnitte etwas umformuliert.